django/contrib/staticfiles/finders.py

```diff
@@ -85,6 +85,14 @@ def check(self, **kwargs):
                     'STATIC_ROOT setting.',
                     id='staticfiles.E002',
                 ))
+        if settings.MEDIA_URL and settings.STATIC_URL and \
+            settings.MEDIA_URL.startswith(settings.STATIC_URL):
```

Comment:
This is the line that caused `flake8` to fail: https://djangoci.com/job/flake8/15588/console